instrumentation/elastic-opentelemetry-instrumentation-openai/src/opentelemetry/instrumentation/openai/helpers.py

```diff
@@ -152,6 +152,10 @@ def _get_attributes_from_wrapper(instance, kwargs) -> Attributes:
         if isinstance(response_format, Mapping):
             if (response_format_type := response_format.get("type")) is not None:
                 span_attributes[GEN_AI_OPENAI_REQUEST_RESPONSE_FORMAT] = response_format_type
+        elif not isinstance(response_format, str):
```

Comment:
Perhaps just a preference for less negatives where possible, it feels easier to read for me if this is the `elif str` case and `else` handles the special one. No worries either way

Reply:
yep much better